sig/datadog/ci/itr/coverage/ddcov.rbs

```diff
@@ -3,8 +3,9 @@ module Datadog
     module ITR
       module Coverage
         class DDCov
+          type threading_mode = :multi | :single
 
-          def initialize: (root: String, ignored_path: String?) -> void
+          def initialize: (root: String, ignored_path: String?, ?threading_mode: threading_mode?) -> void
```

Comment:
Minor: Maybe consider not making this argument optional? (The thinking being: if the production code always sets it, why make it more flexible?)